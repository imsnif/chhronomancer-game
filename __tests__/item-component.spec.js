import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

test('Item component => renders correctly', () => {
  const Item = require('../components/item').default
  const tree = renderer.create(
    <Item fill bright name='assist' />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
