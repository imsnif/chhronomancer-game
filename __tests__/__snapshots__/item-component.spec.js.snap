// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item component => renders correctly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 16,
      "marginRight": 4,
      "width": 16,
    }
  }
/>
`;
